src/Comparable.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ArchTech\Enums;
+
+trait Comparable
+{
+    public function is(mixed $enum): bool
+    {
+        return $this === $enum;
+    }
+
+    public function isNot(mixed $enum): bool
+    {
+        return ! $this->is($enum);
+    }
+
+    public function in(array $enums): bool
+    {
+        return [] !== array_filter($enums, fn (mixed $enum) => $this->is($enum));
+    }
+
+    public function notIn(array $enums): bool
+    {
+        return ! $this->in($enums);
+    }
```

Comment:
Not sure if `is()` and `isNot()` are worth adding.

`in()` and `notIn()` can make sense and then adding the previous two might make sense since it's similar syntax.

Do you have some real world use cases for this?

Reply:
@stancl Yeah, I use them because they are fluent instead of use compare operator such as `===`, `!==`, ... I always use them in my projects. My use case that I want to update the status of blog but some status can be updated. Instead of compare per status, I can use `in` method. I think they are very helpful.

Reply:
@stancl are they ok?